src/features/features.tsx

```diff
@@ -84,6 +86,13 @@ const FEATURES: Features = {
     ui: <AspectRatioUi parents={['ASPECT_RATIO']} />
   },
 
+  SPEED: {
+    name: 'Change Speed',
+    description: 'Change speed of the video',
+    feature: SpeedFeature,
+    ui: <SpeedFeatureUi />
```

Comment:
This should incorporate the parents so that it can used with other features and such https://docs.videotranscode.space/pages/contributing/features/ui.html